src/components/RequestsFilters/PickupServicePointFilter/PickupServicePointFilter.test.js

```diff
@@ -0,0 +1,32 @@
+import '__mock__/';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import PickupServicePointFilter from './PickupServicePointFilter';
+
+const servicePoints = [
+  { id: '1', name: 'Service Point 1' },
+  { id: '2', name: 'Service Point 2' },
+  { id: '3', name: 'Service Point 3' },
+];
+
+describe('PickupServicePointFilter', () => {
```

Comment:
Could you also check that `MultiSelectionFilter` is rendered and called with correct props?